tests/admin/test_controller.py

```diff
@@ -141,7 +148,8 @@ def test_redirect_to_default_library(self):
             response = self.manager.admin_view_controller(None, None)
             eq_(302, response.status_code)
             location = response.headers.get("Location")
-            assert "admin/web/collection/%s" % self._default_library.short_name in location
+            library = Library.default(self._db)
+            assert "admin/web/collection/%s" % library.short_name in location
```

Comment:
Nothing wrong with this code but I don't see why the change is necessary.

Reply:
This is related to the stuff I changed to use nested transactions - it started committing things to the database and broke self._default_library. Once I figure out how to handle that I can change this back.